theme/themes/pxt/elements/loader.overrides

```diff
@@ -15,11 +15,16 @@
   box-shadow: none;
   border-radius: 0px;
   background: transparent @loaderImage no-repeat center center;
-  background-size: 100%;
+  background-size: @loaderImageSize;
  -webkit-animation: @loaderAnimation @loaderSpeed infinite linear;
  animation: @loaderAnimation @loaderSpeed infinite linear;
 }
 
+.ui.loader.main.msft:after {
+  background: transparent @loaderMsftImage no-repeat center center;
```

Comment:
```suggestion
  background-image: @loaderMsftImage;
```

should work fine with a little less duped? doesn't really matter though, either way works